### PR TITLE
[Docs/Website] skipPollingIfUnfocused added to polling overview and query options

### PR DESCRIPTION
**Pull request by riqts**
Added additional note to polling overview and the options in API Reference for the option to pause polling if unfocused

### CODE REVIEW

File: docs/rtk-query/usage/polling.mdx [hunk 1]
```diff
@@ -14,6 +14,10 @@ description: 'RTK Query > Usage > Polling: re-fetching data on a timer'
 
 Polling gives you the ability to have a 'real-time' effect by causing a query to run at a specified interval. To enable polling for a query, pass a `pollingInterval` to the `useQuery` hook or action creator with an interval in milliseconds:
 
+Polling additionally has the ability to skip sending requests while the window is out of focus. To enable this behavior, pass `skipPollingIfUnfocused: true` to the `useQuery` hook or action creator.
```
Comment (EskiMojo14) [inline docs/rtk-query/usage/polling.mdx:17]: would be cool to move this to a tip admonition or something, after the codeblock - along with an example for the sake of it

**Comment by EskiMojo14** [comment]
thanks!

related: https://github.com/reduxjs/redux-toolkit/issues/4118 (may not close it if there are other new features to document)

File: docs/rtk-query/usage/polling.mdx (hunk 1 above)
Comment (riqts) [reply docs/rtk-query/usage/polling.mdx:18]: most recent commit should represent this change. I added it to the existing example with some notes but I can move it to its own as well if preferred.

File: docs/rtk-query/usage/polling.mdx [hunk 2]
```diff
@@ -14,14 +14,21 @@ description: 'RTK Query > Usage > Polling: re-fetching data on a timer'
 
 Polling gives you the ability to have a 'real-time' effect by causing a query to run at a specified interval. To enable polling for a query, pass a `pollingInterval` to the `useQuery` hook or action creator with an interval in milliseconds:
 
+:::tip
+Polling additionally has the ability to skip sending requests while the window is out of focus. To enable this behavior, pass `skipPollingIfUnfocused: true` to the `useQuery` hook or action creator.
+
+_Note: `skipPollingIfUnfocused` requires [`setupListeners`](../api/setupListeners.mdx) to have been called and is disabled by default_
```
Comment (EskiMojo14) [inline docs/rtk-query/usage/polling.mdx:20]:
```suggestion
_Note: `skipPollingIfUnfocused` requires [`setupListeners`](../api/setupListeners.mdx) to have been called._
```

I don't think it needs to be re-stated that it's disabled by default, it's implied by the sentence above saying you need to enable it

Comment (riqts) [reply docs/rtk-query/usage/polling.mdx]: Good point! updated the PR to not have that redundant wording.

**Comment by EskiMojo14** [comment]
thanks!

**Pull request merged**